Inventory.py

```diff
@@ -0,0 +1,40 @@
+import curses
+
+
+class Item(object):
+    def __init__(self, name, heal_value):
+        self.name = name
+        self.heal_value = heal_value
+
+
+class Equipment(object):
```

Comment:
Bitte keine Klassen nur für Datenspeicherung erstellen. Klassen sollten wenigstens eine Methode haben, die nicht der Konstruktor ist. Für reine Datenklassen eignen sich `namedtuple` sehr gut. In diesem Fall würde das so aussehen:
```python
from collections import namedtuple
Equipment = namedtuple('Equipment', ('name', 'strength'))
```